src/components/PageButtons/PageButtons.test.jsx

```diff
@@ -0,0 +1,38 @@
+import renderer from 'react-test-renderer';
+import PageButtons from '.';
+
+const createInput = function createInput(prevPage, nextPage) {
+  return {
+    prevPage,
+    nextPage,
+    selectedTrack: 't',
+    selectedCohort: 'c',
+    getPrevNextGrades() {},
+  };
+};
+
+describe('PageButtons', () => {
+  const assertPageButtonsSnapshot = function assertPageButtonsSnapshot(input) {
+    const pb = renderer.create(PageButtons(input));
+    const tree = pb.toJSON();
+    expect(tree).toMatchSnapshot();
+  };
+
+  describe('render empty links as disabled', () => {
+    it('prev null, next null', () => {
+      assertPageButtonsSnapshot(createInput(null, null));
+    });
+
+    it('prev null, next not null', () => {
+      assertPageButtonsSnapshot(createInput(null, 'np'));
+    });
+
+    it('prev not null, next null', () => {
+      assertPageButtonsSnapshot(createInput('pp', null));
+    });
+
+    it('prev not null, next not null', () => {
```

Comment:
#NIT it would be clearer for the test cases if this is pulled out into it's own test `describe` function